project2/src/utils/index.js

```diff
@@ -0,0 +1,3 @@
+export function FormatMoney(number){
```

Comment:
```suggestion
export function formatMoney(number) {
```
Đừng đặt tên in hoa tất cả mọi thứ như thế chứ em